### DISCUSSION TITLE
Is there a way to store extensions in two separate places?

### DISCUSSION THREAD

**Discussion by yuwei1chen** [coder/code-server#5295] [Q&A] [answered]
Hi,

I have great need to storage my extensions in two different places, storing all builtin extension in one dedicated folder. I see that "extra extensions dir" was removed in 4.0, and "builtin extension dir" does't work according to discussion#5073 (I also tested it myself, the option didn't take effect).

This is really important for my team. Great thanks if anybody can help.

**Comment by jsjoeio** [comment]
I'm not as familiar with the extra extensions dir but maybe @code-asher is

**Comment by code-asher** [comment, marked as answer]
Ah yeah as you noticed there is no way built into code-server to do this anymore.  If the `--builtin-extensions-dir` flag is still not working then the only thing that comes to my mind is to copy your extensions directly into the default builtin directory (`<install-dir>/lib/vscode/extensions`).
